src/main/java/com/celements/web/service/WebUtilsService.java

```diff
@@ -1459,4 +1461,11 @@ public EntityType resolveEntityTypeForFullName(String fullName,
     return componentManager.lookupMap(role);
   }
 
+  private IDocumentParentsListerRole getDocumentParentsLister() {
+    if (docParentsLister == null) {
+      docParentsLister = Utils.getComponent(IDocumentParentsListerRole.class);
```

Comment:
Is the problem with the cyclic dependency resolved? Otherwise please add the comment again.